DDPFF/run/ascii.py

```diff
@@ -0,0 +1,12 @@
+import sys
+import numpy as np
+import open3d as o3d
+
+path = 'input/'+sys.argv[1]
+pcd = o3d.io.read_point_cloud(path)
+
+size = len(pcd.points)
+colors = np.zeros((size,3))
+pcd.colors = o3d.utility.Vector3dVector(colors)
+
```

Comment:
It is a good way to use `if __name__ == "__main__":` construction in all executable python scripts --- read about it and add to all created scripts

Reply:
Done